toolbox/fdc3-conformance/FDC3-1.2-Conformance-Test-Cases.md

```diff
@@ -0,0 +1,190 @@
+
+### User Channels Broadcast (Filtered Context)
+
+| App | Step               | Details                                                                                                                                                              |
+|-----|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| A   | 1. addContextListener | Call `fdc3.addContextListener("fdc3.instrument", handler)`<br>Check listener object returned<br>Check that there is an `unsubscribe` function on the returned object |
+| A   | 2. joinChannel        | `fdc3.getSystemChannels()`<br>Check channels are returned.<br>Call `fdc3.joinChannel()` on first non-global channel                                                  |
+| B   | 3. joinChannel        | `fdc3.getSystemChannels()`<br>Check channels are returned.<br>Call `fdc3.joinChannel()` on first non-global channel                                                  |
+| B   | 4. Broadcast          | `fdc3.broadcast()` the instrument context.<br>`fdc3.broadcast()` a contact context.                                                                                  |
+| A   | 5. Receive Context    | Instrument object matches the one broadcast in 4 above.<br>Check that the contact is not received.                                                                   |
+
+-  `UCFilteredContext1`: Perform above test 
+
+| App | Step               | Details                                                                                                                                                              |
+|-----|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| A   | 1. addContextListener | Call `addContextListener (“fdc3.instrument”, handler)`<br>Check listener object returned<br>Check that there is an unsubscribe function on the returned object<br>Call `addContextListener (“fdc3.contact”, handler)`<br>Check listener object returned<br>Check that there is an unsubscribe function on the returned object                                                                                                                                                                                   |
+| A   | 2. joinChannel        | Check that there is an unsubscribe function on the returned object                                                                                                                                                                                                                                                                                |
```

Comment:
this should replicate the next step (so that you have a channel to join)

```suggestion
| A   | 2. joinChannel        | `fdc3.getSystemChannels()`<br>Check channels are returned.<br>Call `fdc3.joinChannel()` on first non-global channel. Check that there is an unsubscribe function on the returned object                                                                                                                                                                                                                                                                                |
```